### PR TITLE
Final changes before release

### PR DESCRIPTION
**Pull request by rubenhorn**
* Launch captive screen on 2022-02-01
* Design of link on captive screen
* Access settings from captive screen (Not sure about this last commit, since it was commented out previously, but it is in the design and having it makes sense)

### CODE REVIEW

**Review by wizlif — DISMISSED** [review]

**Comment by rubenhorn** [comment]
Thank you for approving @wizlif!
Did you also test a build with these changes so we know if it is ready to release?

File: lib/presentation/home/captive_screen.dart [hunk 1]
```diff
@@ -27,19 +30,41 @@ class CaptivePage extends StatelessWidget {
...
-                        //   mainAxisAlignment: MainAxisAlignment.end,
-                        //   children: [
-                        //     CustomFAB(
-                        //       isMini: true,
-                        //       onPressed: () {},
-                        //       child: const Icon(Icons.settings),
-                        //     )
-                        //   ],
-                        // ),
-                        ContentPlaceholder(
```
Comment (Xazin) [inline lib/presentation/home/captive_screen.dart:42]: I guess this was supposed to open the settings, according to the design of the captive page, there was a 'settings' mini-fab in the top right corner.

Comment (rubenhorn) [reply lib/presentation/home/captive_screen.dart:42]: Yes, but this did not match the design, so I copied the button from the profile page.
Is it not being displayed for you?

Comment (Xazin) [reply lib/presentation/home/captive_screen.dart:42]: Yeah, making some last minute changes and fixing some exceptions from captive screen, will make a commit in a few minutes.

**Review by Xazin — APPROVED** [review]

**Pull request merged**